server/test/processs3.test.js

```
@@ -0,0 +1,13 @@
+import { describe, it, expect } from 'vitest';
+import { processFileFromS3 } from '../llm.js';
+
+describe('processFileFromS3', () => {
+  it('throws when S3 not configured', async () => {
+    const { S3_BUCKET, AWS_REGION } = process.env;
+    delete process.env.S3_BUCKET;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)